bench/src/main/scala/benchmarks/Bench.scala

```diff
@@ -0,0 +1,21 @@
+package benchmarks
+
+import org.openjdk.jmh.annotations._
+
+import chess.PerftTestCase
+
+import java.util.concurrent.TimeUnit
+
+@State(Scope.Thread)
```

Comment:
These configs and numbers are copy from [Daniel's async-benchmark](https://github.com/djspiewak/async-runtime-benchmarks/blob/main/src/main/scala/benchmarks/Benchmarks.scala)